pages/common/azure-func.md

```diff
@@ -0,0 +1,29 @@
+# func
+
+> Azure Functions Core Tools lets you develop and test your functions on your local computer from the command prompt or terminal.
+> Your local functions can connect to live Azure services, and you can deploy a function app to your Azure subscription.
```

Comment:
Perhaps this will read better?
```suggestion
> Azure Functions Core Tools: Develop and test Azure Functions locally.
> Local functions can connect to live Azure services, and can deployed a function app with an Azure subscription.
```